diff --git a/src/SFML/Window/OSX/HIDInputManager.mm b/src/SFML/Window/OSX/HIDInputManager.mm
index 6d31ae6..8e6f8b7 100644
--- a/src/SFML/Window/OSX/HIDInputManager.mm
+++ b/src/SFML/Window/OSX/HIDInputManager.mm
@@ -738,8 +738,8 @@
     // the associate IOHIDElementRef with a sf::Keyboard::Scancode
     // in approximately constant time.

-    // Get only keyboards
-    CFSetRef underlying = copyDevices(kHIDPage_GenericDesktop, kHIDUsage_GD_Keyboard);
+    // Skip obtaining HID devices to prevent triggering input monitoring request on macOS >= 10.15
+    CFSetRef underlying = NULL;
     if (underlying == NULL)
     {
         err() << "No keyboard detected by the HID manager!" << std::endl;
